week4/Q1_test.py

```diff
@@ -0,0 +1,26 @@
+import Q1
+import unittest
+
+
+class IslandCountTest(unittest.TestCase):		
+
+	def testBaseCase(self):
```

Comment:
I think there are a few other interesting cases.
What if the map has no land at all?
What if the map has only land?
You can also have specific tests for interesting cases like:
```
X   
 XXX
```
where you want to verify that the diagonal connection does not count.